4.gene_expression_signatures/scripts/2.download-target-data.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Download TARGET Pan Cancer Data for Compression
+# The notebook downloads gene expression and clinical data from the TARGET project. The data is downloaded from UCSC Xena.
+# The data is in log2(FPKM) RSEM transformed
+
+# In[1]:
+
+
+import os
+from urllib.request import urlretrieve
+import random
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+
+# In[2]:
+
+
+url = 'https://toil-xena-hub.s3.us-east-1.amazonaws.com/download/'
+name = 'target_RSEM_gene_fpkm.gz'
+
+path = os.path.join('data', name)
+
+
+# In[3]:
+
+
+urlretrieve('{}{}'.format(url, name), path)
+
+
+# In[4]:
+
+
+get_ipython().system(" sha256sum 'data/target_RSEM_gene_fpkm.gz'")
```

Comment:
Interesting that this is `sha256` but the previous notebook was `MD5`. Any reason for this? I recommend keeping this consistent when possible.

Reply:
corrected